django/contrib/gis/db/models/fields.py

```diff
@@ -119,7 +123,7 @@ def db_type(self, connection):
     # the spheroid corresponding to the SRID of the BaseSpatialField.
     def _get_srid_info(self, connection):
         # Get attributes from `get_srid_info`.
-        self._units, self._units_name, self._spheroid = get_srid_info(self.srid, connection)
+        self._units, self._units_name, self._spheroid = get_srid_info(self.srid, connection)[:3]
```

Comment:
@claudep 
BTW, don't you think that we should avoid such caching? Thanks for opinion.